salt/top.sls

```diff
@@ -3,6 +3,16 @@
 base:
   '*':
     - core
+    - core.sshd
+    - core.swap
+    - core.mail
+    - core.fail2ban
+    - core.locale
+    - core.ntp
+    - core.motd
+    - core.apt
+    - core.systemd
+    - core.customisation
```

Comment:
Does the order matter for these? If not, alphabetical order is preferred.

Reply:
I've made these alphabetical. Any order that is needed it is defined using the "require" arguments in the state files themselves.